core/domain/exp_jobs_one_off.py

```diff
@@ -377,7 +377,7 @@ def map(item):
                     exp_services.get_multiple_explorations_by_version(
                         exploration.id, versions))
             except Exception as e:
-                yield e
+                yield ('ERROR with exp_id %s' % item.id, str(e))
```

Comment:
Any potential unicode issues here?

Reply:
I _don't_ think so, because this is just exp IDs (and lines of code).